cli/app.go

```diff
@@ -1548,10 +1617,9 @@ var app = &cli.App{
 									Name:  "with-upload",
 									Usage: "submits custom training job with an upload training script on data in Viam cloud",
 									UsageText: createUsageText("train submit custom with-upload",
-										[]string{
-											datasetFlagDatasetID, trainFlagModelOrgID, trainFlagModelName,
-											mlTrainingFlagPath, mlTrainingFlagName, mlTrainingFlagArgs,
-										}, true),
+										[]string{generalFlagOrgID, datasetFlagDatasetID, trainFlagModelOrgID, trainFlagModelName, generalFlagPath, mlTrainingFlagName},
```

Comment:
(q) nice catch on adding the `org-id` flag here! However, I notice we're now no longer including the `args` flag. Is this intentional?